internal/testhelpers/creds.go

```diff
@@ -0,0 +1,123 @@
+package testhelpers
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"gopkg.in/yaml.v3"
+
+	prismgoclient "github.com/nutanix-cloud-native/prism-go-client"
+	"github.com/nutanix-cloud-native/prism-go-client/environment/providers/local"
+)
+
+const (
+	nutanixEndpointDummyValue = "prism-test.nutanix.com"
+	nutanixPortDummyValue     = "9440"
+	nutanixUsernameDummyValue = "test"
+	nutanixPasswordDummyValue = "test"
+	nutanixInsecureDummyValue = false
+	prismDevConfigFileName    = ".prism-dev.yaml"
+	prismDevConfigFileEnvVar  = "PRISM_DEV_CONFIG"
+)
+
+type prismCentral struct {
+	Endpoint              string `yaml:"endpoint"`
```

Comment:
can we remove yaml tags as it wont be used here

Reply:
so, when we do a `yaml.Unmarshal` how will the unmarshaller know which field to unmarshal where?